app/logConfig.py

```diff
@@ -0,0 +1,50 @@
+LOGGING_CONFIG = { 
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': { 
+        'standard_formatter': { 
+            'format': '%(asctime)s [%(levelname)s] %(name)s: %(message)s'
+        },
+        # 'request_formatter': { 
+        #     'format': '%(asctime)s [%(levelname)s] %(name)s: %(type)s : %(clientip)s : %(endpoint)s : %(message)s'
+        # }
+    },
+    'handlers': { 
+        # 'req_handler': { 
+        #     'level': 'INFO',
+        #     'formatter': 'request_formatter',
+        #     'class': 'logging.handlers.RotatingFileHandler',
+        #     'filename': 'customlogs.log',
+        #     'mode': 'a',
+        #     'maxBytes': 20,
+        #     'backupCount': 0
+        # },
+        'standard_handler':{
+            'level': 'DEBUG',
+            'formatter': 'standard_formatter',
+            'class': 'logging.handlers.RotatingFileHandler',
+            'filename': 'serverlog.log',
```

Comment:
Can you put the serverlog.log into a folder called logs? 
Then plz add logs dir to .gitignore, so we don't push that my mistake after any server run.

Reply:
Sure. Will add it.